packages/devtools-client-adapters/src/chrome.js

```diff
@@ -78,6 +78,10 @@ function connectNode(tab) {
   return CDP({ tab: tab.webSocketDebuggerUrl })
     .then(conn => {
       connection = conn;
+      window.addEventListener("beforeunload", () => {
+        connection.onclose = function disable() {};
```

Comment:
In researching this I found that others hadn't disabled the `onclose` handler and then other function would try to run, but in this case we are out of time and the window is closing.